src/mlpack/core/data/dataset_mapper_impl.hpp

```diff
@@ -133,15 +130,14 @@ inline Datatype DatasetMapper<PolicyType>::Type(const size_t dimension) const
 template<typename PolicyType>
 inline Datatype& DatasetMapper<PolicyType>::Type(const size_t dimension)
 {
-  if (dimension >= types.size())
-    types.resize(dimension + 1, Datatype::numeric);
+  if (dimension >= types.size()) types.resize(dimension + 1, Datatype::numeric);
```

Comment:
Yes, fixed this by setting `AllowShortIfStatementsOnASingleLine` in the configuration file to false.